scripts/globals/mobskills/eternal_damnation.lua

```diff
@@ -0,0 +1,19 @@
+---------------------------------------------
+-- Eternal Damnation
+-- Description: Inflicts Doom upon an enemy.
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/status")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local typeEffect = dsp.effect.DOOM
```

Comment:
no need to var this, its unchanging and doesn't involve a function call. it just evals to a number and its already loaded.